features/other_b/series.feature

```diff
@@ -78,112 +76,79 @@ Feature: Create and Edit Series
       And I check "series_complete"
       And I press "Update"
     Then I should see "Complete: Yes"
+
+  @disable_caching
+  Scenario: A work can be in two series
+    Given I am logged in as "author"
+      And I post the work "Sweetie Belle" as part of a series "Ponies"
+      And I post the work "Rainbow Dash" as part of a series "Ponies"
     When I edit the work "Rainbow Dash"
     Then the "series-options-show" checkbox should be checked
       And I should see "Ponies" within "fieldset#series-options"
     When I fill in "work_series_attributes_title" with "Black Beauty"
       And all search indexes are updated
       And I press "Preview"
-    Then I should see "Part 3 of the Ponies series" within "dd.series"
-    # TODO fix issue 3461 (manifests when perform_caching: true)
+    Then I should see "Part 2 of the Ponies series" within "dd.series"
+    When "AO3-3455" is fixed
       # And I should see "Part 1 of the Black Beauty series" within "dd.series"
     When I press "Update"
       And all search indexes are updated
     Then I should see "Part 1 of the Black Beauty series" within "dd.series"
-      And I should see "Part 3 of the Ponies series" within "dd.series"
+      And I should see "Part 2 of the Ponies series" within "dd.series"
       And I should see "Part 1 of the Black Beauty series" within "div#series"
-      And I should see "Part 3 of the Ponies series" within "div#series"
+      And I should see "Part 2 of the Ponies series" within "div#series"
 
-  @disable_caching
-  Scenario: Three ways to add a work to a series: a user with more than one pseud
-    Given the following activated user exists
-      | login         | password   |
-      | author        | password   |
-      And a warning exists with name: "Choose Not To Use Archive Warnings", canonical: true
-      And the default ratings exist
-    When I am logged in as "author" with password "password"
-      And "author" creates the default pseud "Pointless Pseud"
-    When I go to the new work page
-      And I select "Not Rated" from "Rating"
-      And I check "Choose Not To Use Archive Warnings"
-      And I fill in "Fandoms" with "My Little Pony"
-      And I select "Pointless Pseud" from "work_author_attributes_ids_"
-      And I check "series-options-show"
-      And I fill in "work_series_attributes_title" with "Ponies"
-      And I fill in "Work Title" with "Sweetie Belle"
-      And I fill in "content" with "First little pony is all alone"
+  Scenario: Creator with multiple pseuds adds a work to a new series when the work is first posted
+    Given I am logged in as "author"
+      And I add the pseud "Pointless Pseud"
+      And I set up the draft "Sweetie Belle" using the pseud "Pointless Pseud"
```

Comment:
Again I can't see why this is not 

And I set up the draft "Sweetie Belle" using the pseud "Pointless Pseud"  as part of a series "Ponies"